README.md

```diff
@@ -1,3 +1,8 @@
 # Drive Science Android Demo App
 
-This app is a demonstration of how you might integrate with the Android Trip Tracker SDK. For further documentation on the integration process, please refer to the [Android docs](https://github.com/Root-App/drive-science-docs/blob/master/android.md).
\ No newline at end of file
+This app is a demonstration of how you might integrate with the Android Trip Tracker SDK. For further documentation on the integration process, please refer to the [Android docs](https://github.com/Root-App/drive-science-docs/blob/master/android.md).
```

Comment:
Not a change for this PR, but why does this not live in [readme like our ios docs](https://developer.root-enterprise.com/docs/ios) do?

The repo you linked to even says it is deprecated as a resource.

![image](https://user-images.githubusercontent.com/58446296/100627668-e294b800-32ec-11eb-8f27-8e2d956d0f3a.png)